time.go

```diff
@@ -29,9 +25,13 @@ func (t Time) MarshalJSON() ([]byte, error) {
 }
 
 func (t Time) String() string {
+	return time.Time(t).String()
+}
 
-	if time.Time(t).IsZero() {
-		return ""
+func (t Time) QueryValue() string {
+	ts := time.Time(t).UnixNano() / 1e6
```

Comment:
If time.Time(t).IsZero(), I think we should return an empty string (i.e. unspecific time).

Otherwise, I think it's ok just to strconv.FormatInt(). If someone passes in time.Time{}.Add(time.Second) (i.e. 0001-01-01 00:00:01), then let's submit a negative number. Don't feel strongly about that though - returning "0" is also ok.

Reply:
I don't feel strong for this PR at all:)
I thought negative url should be fine at the beginning, or we should have change the interface for  string at exchange.
Anyway. What you are suggesting is least evil. ie, negative time should be fine for all but zero time[which should be empty string, not an int]. let me amend it